### DISCUSSION TITLE
The Home Tab Enabled in Version 3.5.2 by default

### DISCUSSION THREAD

**Discussion by XJJ76** [sumatrapdfreader/sumatrapdf#3818] [General]
The Home tab is enabled by default. This is not convenient when open a pdf file. This issue doesn't happen at the previous version.
The solution is set “NoHomeTab = true” by default in the SumatraPDF-settings.txt.

**Comment by GitHubRulesOK** [comment]
I can be wrong however
AFAIK, **when it was introduced,** the Home Tab setting  default was always "false" and user has to opt out.

**Comment by kjk** [comment]
This is how I want it to be so it won't change.

**Reply by navossoc** [reply]
Sad to hear that...

I also don't have anything on the home tab, but do as @jumper444 suggested, close the tab when opening a document.

It's horrible that you click to open 3 pdf files then, try to ctrl+tab between them and have a "home page" between the documents.

It really doesn't make any sense.

**Reply by GitHubRulesOK** [reply]
@navossoc 
If you dont want "HOME" tab simply switch it off no need to close it
`NoHomeTab = true`

**Reply by navossoc** [reply]
@GitHubRulesOK 
After some trial and error. I got it.
It works as expected if `NoHomeTab = true`.

Make sure you double check if there is no other `NoHomeTab` settings lying around on the .txt file.

**Comment by jumper444** [comment]
I think there might be a bit of confusion here and I think the issue is legitimate if I may explain.

When Sumatra(program) is opened normally/directly, it creates a blank 'home' tab (which has the version and links to website, etc.)  This is expected and is similar to how Excel or Word open with a new blank document.  If Sumatra is opened as a program having a blank starting document it makes sense. (Word and Excel dont put text or links or version numbers in their starting document/spreadsheet, but a PDF is not a typable/editing document so what Sumatra is doing is fine.)

However when Windows opens a pdf through association (from Explorer,etc) Sumatra is NOT opening blank and without a file (which would justify a "Home" tab - just like Excel and Word have a new blank document). Instead it now has a PDF to display and I agree that the program should only be displaying this PDF and only have a single tab (not *two* tabs making it look like two documents are open:  "Home" and "my PDF").

This is the same as Excel or Word or most other programs - if you open them through association by launching a document they do not open with TWO documents (aka a blank/new/home/untitled document *AND* the document you just opened from Explorer), they just open the one (launched by association) file.

Expected behavior in my opinion:
-If Sumatra is launched directly it should open with blank/new "Home" tab (which can have version and links or whatever).
-If Sumatra is launched by file association it should have ONLY that file as a tab and the blank/new "Home" tab should not be there.  (If a person needs Sumatra version or links they can go to Help->About)

This behavior mirrors behavior of most other programs I know (all to be honest) and does not result in a new program launch looking like it immediately has TWO documents (which is the current problem with Sumatra).

I also think this behavior suggested (like other programs) is DIFFERENT from “NoHomeTab = true” and I agree that would not be the solution.  You DO want the Home tab/page on a (blank/direct) program launch.  You just dont want it when the program launch is via a file association and thus Sumatra has a document to display.

**Comment by jumper444** [comment]
Attached screenshot. If I double click a PDF and it opens in Sumatra, why should the program immediately look like it has TWO files open.  This confuses (and casues me to repeatedly glance and think 'wait...oh..yeah that's the blank home tab that always opens').  So I have to mentally train myself to realize I dont have an additional document there that needs attention (or i have to close it each time to get it out of the way, but it will appear again next time).  I dont think this is the proper behavior, for your consideration.
![sumatra2tabs](https://github.com/sumatrapdfreader/sumatrapdf/assets/1995883/2304ac0c-0f4b-4f3a-9593-d5cb444df817)

**Reply by kjk** [reply]
Set `NoHomeTab = true` advanced setting and it will work as you expect it.

**Reply by navossoc** [reply]
Correction: `NoHomeTab = true`

**Reply by kjk** [reply]
yes, sorry, I mistyped and updated my comment above

**Comment by jumper444** [comment]
Interestingly, if i close the document tab above ("Tax-recovery....") then **both** that tab and the "Home" tab close and i end up with no tabs as shown.  You would think the "Home" tab would stay but it isn't.  Unexpected behavior.

![sumatraClosed1](https://github.com/sumatrapdfreader/sumatrapdf/assets/1995883/ef8c5938-5eb7-4730-a597-4b1d238db78b)

**Reply by kjk** [reply]
It's as I want it to work. No point showing a tab if there's only one shown. It's a cleaner look.

**Comment by jumper444** [comment]
Note that this "NoHomeTab = true" option is not documented:
https://www.sumatrapdfreader.org/settings/settings3-4-6

EDIT: Sorry, I see there is a new settings page (and it is on there):
https://www.sumatrapdfreader.org/settings/settings
